source/screens/App/components/AppEntry.js

```diff
@@ -1,30 +1,33 @@
 import React from 'react'
-import {
+import { 
   BrowserRouter as Router,
   Switch,
-  Route,
+  Route
 } from 'react-router-dom'
+
 import Logo from './Logo'
+import { SideMenuBar } from '../components'
+import { Home } from '../screens/Home'
 import { Jobs } from '../screens/Jobs'
 import { Leads } from '../screens/Leads'
 import { Materials } from '../screens/Materials'
 import { Quotes } from '../screens/Quotes'
-import { UserProfile } from '../screens/UserProfile'
 import { Vendors } from '../screens/Vendors'
 import './AppEntry.css'
 
 const AppEntry = () => (
   <div className="AppEntry">
     <div className="AppEntry-Menu">
-      <Logo className="AppEntry-Logo" />
+      <Logo className="AppEntry-Logo"/>
+      <SideMenuBar />
     </div>
     <div className="AppEntry-Body">
       <Router>
         <Switch>
+          <Route exact path="/" component={Home} />
           <Route exact path="/jobs" component={Jobs} />
           <Route exact path="/leads" component={Leads} />
           <Route exact path="/materials" component={Materials} />
-          <Route exact path="/profile" component={UserProfile} />
           <Route exact path="/quotes" component={Quotes} />
```

Comment:
Same question as above, not a big deal, I can add it back in, just don't want to lose it

Reply:
I'll add it. let's go through it together.